gateway-js/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the appropriate changes within that release will be moved into the new section.
 
+- __BREAKING__: Make all `protected` gateway fields `private`. [PR #539](https://github.com/apollographql/federation/pull/539)
```

Comment:
Change "fields" to "fields and methods" in this changelog entry and the PR description?

Reply:
Also could expand a bit more like "we don't believe we've ever documented subclassing ApolloGateway as an encouraged way to use it; if you rely on subclassing ApolloGateway and accessing protected members, let us know"?